### DISCUSSION TITLE
Keyboard Jogging - Numeric Keypad versus Regular Numbers for modifying Distance/Speed

### DISCUSSION THREAD

**Discussion by KingBubbaTruck** [terjeio/ioSender#315] [General]
Loving IoSender and Using Phils BOB Unkit.  Really working well for me.

I have yet to wrap my head around how to get a pendant working, but I have a small wireless keyboard with a trackpad that I've used in the past for jogging.  It's got a little set of arrow buttons and pgup/pgdown that work well with IoSender.  What it does not have is a numeric keypad so I'm not able to change the step size or step speed without going to my computer to change the settings either on the full size keyboard or the ui.

Is there a way I can change things up so I can use the <Ctrl>+<Number> to change the step size and or step speed?

**Comment by terjeio** [comment]
You can save a keymappings file from the _Settings: App_: tab. This can be edited to change the mappings.
The file is saved in the sender directory and is named _KeyMap\<n\>.xml_ where \<n\> is the current [jogmode](https://github.com/terjeio/ioSender/wiki/Setup-and-configuration).

**Reply by KingBubbaTruck** [reply]
Awesome.  Thanks!

**Comment by KingBubbaTruck** [comment]
That worked great.  I wasn't able to use the numeric keys, so I used ASDF for the Jog Distance and ZXCV for the feed rate.

Thank you so much!
